components/definition/definition.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="definition js-details"
+  data-hide-label="{{definitionText}}"
+  data-show-label="{{definitionText}}">
+
+  <a href="#definition"
+     class="definition__link mars js-details-trigger js-details-label u-no-js-hide"
```

Comment:
Can this have analytics event tracking added to the link trigger to capture opening/closing usage and add the analytics to the button to close?

Reply:
I'm not too sure if this is the place to demonstrate Google Analytics working with the module. The version of the Analytics module that traverses the DOM to attach behaviour will work with any other component with just adding HTMl attributes but have included it demonstration purposes